library/alloc/src/collections/vec_deque/mod.rs

```diff
@@ -168,7 +168,7 @@ impl<T, A: Allocator> VecDeque<T, A> {
     ///
     /// May only be called if `deque.len() < deque.capacity()`
     #[inline]
-    unsafe fn push_unchecked(&mut self, element: T) {
+    const unsafe fn push_unchecked(&mut self, element: T) {
```

Comment:
This function is internal-only and doesn't seem to need to be `const` to support public API. Please drop changes to anything that isn't strictly necessary to support making the public functions here `const`, to make it clear exactly what we're (unstably) committing to here.